tekton/cronjobs/README.md

```diff
@@ -203,19 +203,25 @@ patchesStrategicMerge:
 nameSuffix: "-myimage"
 ```
 
-1. Apply your new job:
+5. Edit `tekton/cronjobs/dogfooding/images/kustomization.yaml`. Add name of your newly created directory
```

Comment:
NIT: I prefer using `1.` and let the render set the numbers

Reply:
oops, sorry, it was not my intention to change some working stuff. I just see in my browser only `1.` for all items :(

Reply:
oh, then thanks for fixing it!! 🙏 
I wonder why it's not rendered properly, but I'm not going to try and find out :P